include/fcl/narrowphase/detail/convexity_based_algorithm/epa.h

```diff
@@ -52,9 +52,9 @@ namespace detail
 //static const size_t EPA_MAX_ITERATIONS = 255;
 // TODO(JS): remove?
 
-/// @brief class for EPA algorithm
+/// @brief class FCL_VISIBLE for EPA algorithm
```

Comment:
There are a few spurious insertion of the `FCL_VISIBLE` macro in the documentation, they can be easily found by searching for `FCL_VISIBLE for`.